sherlock/resources/data.json

```diff
@@ -65,6 +65,21 @@
     "username_claimed": "blue",
     "username_unclaimed": "noonewouldeverusethis7"
   },
+  "Anilist": {
+    "errorType": "status_code",
+    "regexCheck": "^[A-Za-z0-9]{2,20}$",
+    "request_method": "POST",
+    "request_payload": {
+      "query": "query($name:String){User(name:$name){id}}",
+      "variables": {
+        "name": "{}"
+      }
+    },
+    "url": "https://graphql.anilist.co/",
```

Comment:
Shouldn't this be `https://anilist.co/user/{user}/` ?

Reply:
`https://graphql.anilist.co` should be `urlProbe`

Reply:
Ahh, sorry about that! Should be good now.